test/abap_transpile.json

```diff
@@ -95,6 +95,9 @@
       {"object": "ZCL_ABAPGIT_FILE_STATUS", "class": "ltcl_calculate_status", "method": "deleted_remote", "note": "READ TABLE subrc 4 vs subrc 8"},
       {"object": "ZCL_ABAPGIT_FILE_STATUS", "class": "ltcl_calculate_status", "method": "complete", "note": "READ TABLE subrc 4 vs subrc 8"},
 
+      {"object": "ZCL_ABAPGIT_FILENAME_LOGIC", "class": "ltcl_run_checks", "method": "file_to_object_package", "note": "Depends on ZCL_ABAPGIT_OBJECT_DEVC, db access"},
+      {"object": "ZCL_ABAPGIT_FILENAME_LOGIC", "class": "ltcl_run_checks", "method": "object_to_file_package", "note": "Depends on ZCL_ABAPGIT_OBJECT_DEVC, db access"},
+
```

Comment:
hmm, these unit tests worked before the change? does the tests pass when running in a real system? and there is no $PACK in the system?

Reply:
fixed with https://github.com/abapGit/abapGit/pull/6235/commits/b6304aa9c7f18e2436700924ea9d2b1f863612f2